tests/unit_tests/data_steward/deid/parser_test.py

```diff
@@ -63,7 +63,7 @@ def test_parse_args(self):
 
         # setting correct_parameter_dict values not set in setUp function
         correct_parameter_dict['cluster'] = False
-        correct_parameter_dict['age-limit'] = 89
+        correct_parameter_dict['age_limit'] = 89
```

Comment:
Would this need to use the constant as well?

Reply:
It should.  I updated this.  Thanks for the catch.